prometheus_client/core.py

```diff
@@ -31,7 +31,7 @@
 _INITIAL_MMAP_SIZE = 1 << 20
 
 _pack_integer = struct.Struct(b'i').pack_into
```

Comment:
Is there the same issue with this?

Reply:
Didn't test in details but I would assume so. I'll apply the same fix there.